Language/id/Errors.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/**
+ * This file is part of CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+// Errors language settings
+return [
+    'pageNotFound'    => '404 - Halaman tidak ditemukan',
+    'sorryCannotFind' => 'Maaf! Sepertinya tidak dapat menemukan halaman yang Anda cari.',
+    'whoops'          => 'Upps!',
+    'weHitASnag'      => 'Kami menemukan adanya halangan, Silahkan coba kembali nanti...',
```

Comment:
Typo: silahkan -> silakan

```suggestion
    'weHitASnag'      => 'Kami menemukan adanya halangan, silakan coba kembali nanti...',
```

Reply:
Done